Print Fibonacci Series

delvein opened the pull request:
Fixes # 24

#### Changes proposed in this PR:

- Added a file to calculate and print fibonacci numbers

delvein committed:
testing

delvein committed:
Adding a file to print fibonacci series as an output

delvein committed:
Deleting this empty file

anzdaddy commented on 01_fib_series/fibseries.go:
```diff
@@ -0,0 +1,20 @@
+	"strconv"
+)
+
+func FibonacciRecursion(n int) int {
```
The first recursive version! 👍 Bonus points if you know the time complexity.

delvein replied:
Since the FibonacciRecursion is called twice within i would think it's time complexity is O(2^n). If this recursion is called only once with n-1, i would say it's O(n) - linear and when the function is called within second time with n-2 is also called linear (actually called order of n/2 times. And, O(n/2) = O(n).
But because it's called twice combined, the answer would be "exponential". Please let me know if this is correct.

juliaogris commented on 01_fib_series/fibseries.go:
```diff
@@ -0,0 +1,20 @@
+package main
```
Tests missing.

delvein replied:
So added testing and have updated all review comments.

juliaogris reviewed (commented):
Please add PR description according to [template](https://github.com/anz-bank/go-training/blob/master/.github/pull_request_template.md).

Please also fix up commit messages with `git rebase -i HASH` according to [recommendations](https://chris.beams.io/posts/git-commit/):
* Capitalize the subject line
* Use the imperative mood in the subject line
* Fold fixup commits into previous ones with `git rebase -i`

delvein committed:
Added a test for fibonacci series printing and corrected few mistakes in fibonacci program

delvein committed:
Merging from upstream to my fork
Merge branch 'master' of https://github.com/anz-bank/go-training into fib-series

delvein committed:
Add a file to bubble sort the given number

juliaogris commented:
-   Click `Update branch` and locally `git pull origin fib-series` before adding new commits.
-   Fix broken build
-   Please edit your commits to follow [recommendations](https://chris.beams.io/posts/git-commit/)
    -   Capitalize the subject line
    -   Use the imperative mood in the subject line
    -   Fold fixup commits into previous ones with `git rebase -i HASH`

juliaogris commented:
Hi,
There has been  no activity in the last 20 days so I will close this PR.
Feel free to open it again if you have something to add.
Thanks!
Julia
